tests/unit/App.spec.ts

```diff
@@ -1097,3 +1100,53 @@ describe('Restoration - App page', () => {
     expect(wrapper.findComponent(Actions).exists()).toBe(true)
   })
 })
+
+describe('Breadcrumbs for firms', () => {
+  it('computes breadcrumbs correctly for a SP registration', () => {
+    const wrapper = shallowWrapperFactory(
```

Comment:
I decided to shallow-mount App.vue (meaning the Breadcrumb sub-component is stubbed, not fully rendered) and check the `breadcrumbs` computed instead.